assets/app/styles/_core.less

```diff
@@ -388,6 +388,11 @@ select:invalid {
   }
 }
 
+.spinner.spinner-xs { // set height & width to 12px so that FF displays correctly
```

Comment:
can you open an issue against patternfly with this fix, add a comment with a link to the issue here, and add a TODO to the "Update patternfly" issue so we can track removing our workaround?